lms/envs/devstack.py

```diff
@@ -228,3 +228,15 @@ def should_show_debug_toolbar(_):
 MODULESTORE = convert_module_store_setting_if_needed(MODULESTORE)
 
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+
+#####################################################################
+# Mobile
+FEATURES['ENABLE_COURSE_BLOCKS_NAVIGATION_API'] = True
+FEATURES['ENABLE_RENDER_XBLOCK_API'] = True
```

Comment:
oh. nvm.  I forgot my changes for removing the `ENABLE_RENDER_XBLOCK_API` is in a separate PR.   Yeah, you'll need that for now.
But `ENABLE_COURSE_BLOCKS_NAVIGATION_API` shouldn't be needed for now, unless comparing with the old API.